src/app/page/Home/VolunteerHome.jsx

```diff
@@ -101,6 +102,17 @@ export default function VolunteerHome({ currentUser }) {
     updateInProgressMissions(inProgressMissions.filter((m) => m.uid !== missionUid));
   };
 
+  const handleUpdatedMissions = () => {
```

Comment:
a better way of doing this would be, 
do the request in the button, and then refetch what need to refetch, to  explicit pass down this function to the component make the component depends on this props, and can not be reused

Reply:
Ok.  The reason I did that was because after clicking those buttons it changes the state in the top level component (VolunteerHome) and I couldn't figure out how to get a reference to the VolunteerHome's states without passing the function down the button.  But I'll research this more and find a more elegant way.